spec/math_spec.rb

```diff
@@ -343,4 +366,23 @@
       end
     end
   end
+
+  context "#solve", focus: true do
```

Comment:
I'll keep that in mind and fix that in a later commit.

Tired of rebasing now..